docs/pages/repo/docs/getting-started/add-to-project.mdx

```diff
@@ -78,70 +65,70 @@ For more information on configuring your `turbo.json`, see the [Configuration Op
 }
 ```
 
-Some Vite starters ship with a `package.json` that looks like this:
+   Some Vite starters ship with a `package.json` that looks like this:
 
-```json filename="package.json"
-{
-  "scripts": {
-    "build": "tsc && vite build"
-  }
-}
-```
+   ```json filename="package.json"
+   {
+     "scripts": {
+       "build": "tsc && vite build"
+     }
+   }
+   ```
 
-We recommend splitting these into a `lint` and `build` script.
+   We recommend splitting these into a `lint` and `build` script.
```

Comment:
Let's use `type-check` instead of lint here. `tsc` is technically more linter-ish but TypeScript-land identifies it as a type checker. There are other places (either in the examples or somewhere else in the docs) where we're using `type-check` so also in the interest of staying consistent.

Reply:
done